lib/project/cordova-project.ts

```diff
@@ -25,8 +25,8 @@ export class CordovaProject extends FrameworkProjectBase implements Project.IFra
 		$fs: IFileSystem,
 		$jsonSchemaValidator: IJsonSchemaValidator,
 		$logger: ILogger,
-		$options: IOptions,
-		$resources: IResourceLoader) {
+		$resources: IResourceLoader,
```

Comment:
I wanted the order here to be the same as the order in `NativeScriptProject`. This way it will be easier to edit if we add new parameter in the constructor of `FrameworkProjectBase` :smile: